src/main/java/uk/gov/hmcts/darts/usermanagement/component/validation/impl/SecurityGroupCreationValidation.java

```diff
@@ -4,20 +4,20 @@
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.darts.common.component.validation.Validator;
 import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.model.SecurityGroupModel;
 import uk.gov.hmcts.darts.common.repository.SecurityGroupRepository;
 import uk.gov.hmcts.darts.usermanagement.exception.UserManagementError;
-import uk.gov.hmcts.darts.usermanagement.model.SecurityGroup;
 
 import java.util.Collections;
 
 @Component
 @RequiredArgsConstructor
-public class SecurityGroupCreationValidation implements Validator<SecurityGroup> {
+public class SecurityGroupCreationValidation implements Validator<SecurityGroupModel> {
 
     private final SecurityGroupRepository securityGroupRepository;
 
     @Override
-    public void validate(SecurityGroup securityGroup) {
+    public void validate(SecurityGroupModel securityGroup) {
         String name = securityGroup.getName();
 
         securityGroupRepository.findByGroupName(name)
```

Comment:
Wondering if this needs to be case insensitive like the courthouse `findByCourthouseNameIgnoreCase`?

Reply:
Sounds reasonable, updated.